src/zjs_gpio.c

```diff
@@ -325,12 +379,12 @@ static jerry_value_t zjs_gpio_open(const jerry_value_t function_obj,
         // Zephyr ISR callback init
```

Comment:
I'm interested whether handles can be made unconditional so lookup_pin() can be inlined into the open function and all other usages grab the struct *device from the handle.

On a similar vein, it is possible to replace the attributes of the pinobj with "getter" methods? The current approach looks deeply inefficient in terms of RAM usage. If the handle were unconditional then getter methods could integate it and RAM usage should be smaller (not to mention that for bi-directional bit banging we need the "direction" to be hooked up to a setter method anyway).